src/test/java/org/opentripplanner/transit/model/TransitModelArchitectureTest.java

```diff
@@ -49,7 +50,7 @@ void enforcePackageDependencies() {
       )
       .verify();
     TIMETABLE
-      .dependsOn(UTILS, FRAMEWORK, BASIC, ORGANIZATION, NETWORK, SITE, LEGACY_MODEL)
+      .dependsOn(UTILS, FRAMEWORK, BASIC, ORGANIZATION, NETWORK, SITE, LEGACY_MODEL, COMMON_MODEL)
```

Comment:
@t2gran could you check if this is the correct location for the `Result` type

Reply:
Move to `package org.opentripplanner.transit.model.framework`

Reply:
Done.